SofaKernel/modules/SofaMeshCollision/src/SofaMeshCollision/TriangleLocalMinDistanceFilter.cpp

```diff
@@ -311,7 +312,8 @@ bool TriangleLocalMinDistanceFilter::validPoint(const Index pointIndex, const de
 
 bool TriangleLocalMinDistanceFilter::validLine(const Index lineIndex, const defaulttype::Vector3 &PQ)
 {
-    LineInfo &Li = m_lineInfo[lineIndex];  // filter is precomputed
+    helper::WriteAccessor< Data<sofa::helper::vector<LineInfo> > > lInfo(m_lineInfo);
```

Comment:
`auto lInfo = sofa::helper::getWriteAccessor(m_lineInfo)` is clearer

Reply:
```TriangleLocalMinDistanceFilter.cpp(317,35): error C2676: binary '[': 'sofa::helper::WriteAccessor<sofa::component::topology::TopologyData<sofa::topology::Edge,sofa::type::vector<sofa::component::collision::LineInfo,sofa::type::CPUMemoryManager<T>>>,void>' does not define this operator or a conversion to a type acceptable to the predefined operator
12>        with
12>        [
12>            T=sofa::component::collision::LineInfo
12>        ]
12>C:\projects\sofa-src\SofaKernel\modules\SofaMeshCollision\src\SofaMeshCollision\TriangleLocalMinDistanceFilter.cpp(317,18): error C2530: 'Li': references must be initialized
```

Reply:
Ah, too bad; no choice to keep as it is then.